<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui_Registro</class>
 <widget class="QDialog" name="Gui_Registro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuevo Registro</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 127);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 127);</string>
   </property>
   <property name="title">
    <string>Registro Hospedaje Formulario</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ID-Registro</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar Habitaci√≥n</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ID-Cliente</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cliente</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Empleado</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha Ingreso</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha Salida</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="fechaE">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="fechaS">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Registrar_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 184, 62);</string>
    </property>
    <property name="text">
     <string>GUARDAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Cancelar_button">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 184, 62);</string>
    </property>
    <property name="text">
     <string>CANCELAR</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Cliente_cmbox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_idRegis">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdite_idCliente">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_NomEmpleado">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Hab_Reg_Button">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#Hab_Reg_Button{
	background-color: transparent;
	border-image: url(:/imagenes/imagenes/agregar-h_unpressed.png);
}

#Hab_Reg_button:pressed{
	background-color: transparent;
	border-image: url(:/imagenes/imagenes/agregar-h_pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Registrar_button_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 184, 62);</string>
    </property>
    <property name="text">
     <string>VER FACTURA</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
